components/sidebar/add_channel_dropdown.tsx

```diff
@@ -47,6 +45,90 @@ const AddChannelDropdown = ({
     canCreateCustomGroups,
 }: Props) => {
     const intl = useIntl();
+    const dispatch = useDispatch();
+
+    const fallbackFocus = document.activeElement;
```

Comment:
Similar to my comment on the `ModalController`, this should also be stored when the dropdown is opened so that it doesn't change any other time this dropdown re-renders